bob-common/src/configs/node.rs

```diff
@@ -644,6 +647,14 @@ impl NodeConfig {
         AuthenticationType::None
     }
 
+    fn default_hostname_resolve_dur_ms() -> u64 {
+        5000
+    }
+
+    pub fn hostname_resolve_dur_ms(&self) -> u64 {
```

Comment:
`dur` (duration) is not a good name, as it is actually a period. `hostname_resolve_period_ms` is better